data/mods/Magiclysm/effects/effects.json

```
@@ -300,5 +300,18 @@
     "int_add_val": 1,
     "base_mods": { "speed_mod": [ -25 ] },
     "scaling_mods": { "speed_mod": [ -5 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "cold_heart_effect",
+    "name": [ "Cold Heart" ],
+    "desc": [ "Heart of this creacure was shattered, his life slowly slips away." ],
```

This description is more suitable for a direct damage effect.

---

im really bad in descriprions, so i made draft request with hope there will be person that create something more "natural"

---

Well, I'm horrible with coming up with names for things, so I can sympathize... 
```suggestion
    "desc": [ "As the blood chills, the heart slows with its growing cold." ],
```